aten/src/ATen/native/cuda/ForeachPointwiseOp.cu

```diff
@@ -29,6 +29,47 @@ std::vector<Tensor> foreach_pointwise_op(TensorList input, TensorList tensors1,
     return tensor_lists[3];
 }
 
+template<template<class> class Op>
+void foreach_pointwise_op_(TensorList input, TensorList tensors1, TensorList tensors2, at::ArrayRef<double> scalars) {
+    std::vector<std::vector<at::Tensor>> tensor_lists;
```

Comment:
nit: It's good practice to reserve space especially if we know how large the lists will be (in this case, size 3)